src/validation.cpp

```diff
@@ -1439,15 +1455,16 @@ PackageMempoolAcceptResult MemPoolAccept::AcceptPackage(const Package& package,
     }
     // Validate the (deduplicated) transactions as a package. Note that submission_result has its
     // own PackageValidationState; package_state_quit_early is unused past this point.
-    auto submission_result = AcceptMultipleTransactions(txns_package_eval, args);
+    auto submission_result = AcceptPackageWrappingSingle(txns_package_eval);
     // Include already-in-mempool transaction results in the final result.
     for (const auto& [wtxid, mempoolaccept_res] : results_final) {
         Assume(submission_result.m_tx_results.emplace(wtxid, mempoolaccept_res).second);
-        Assume(mempoolaccept_res.m_result_type != MempoolAcceptResult::ResultType::INVALID);
+        Assume(mempoolaccept_res.m_result_type != MempoolAcceptResult::ResultType::INVALID ||
+               mempoolaccept_res.m_state.GetResult() == TxValidationResult::TX_MISSING_INPUTS);
```

Comment:
879b55fa9da3688fb6b85e7f40f9778753a9102f

According to this change, it should now be possible to hit `mempoolaccept_res.m_result_type == VALID/MEMPOOL_ENTRY/DIFFERENT_WITNESS`, but only when [...].

However, I don't see how the `AcceptPackageWrappingSingle` code change touches `mempoolaccept_res` at all. So it's unclear why the Assume is changed here.

Reply:
Oops yes, I've removed this